storage/backup/backup-cli/src/coordinators/backup.rs

```diff
@@ -255,7 +258,7 @@ impl BackupCoordinator {
     ) -> impl StreamExt<Item = ()> + 'a
     where
         S: Copy + Debug + 'a,
-        W: Worker<'a, S, Fut> + Copy + 'static,
+        W: Worker<'a, S, Fut> + Copy + 'a,
```

Comment:
no idea what you are doing here.

Reply:
Now there's a closure passed in, so not always `'static` any more..